gradient/constants.py

```diff
@@ -56,9 +56,14 @@ class Region(object):
     AMS1 = "Europe (AMS1)"
 
 
-MACHINE_TYPES = ("Air", "Standard", "Pro", "Advanced", "GPU+",
-                 "P4000", "P5000", "P6000", "V100",
-                 "C1", "C2", "C3", "C4", "C5", "C6", "C7", "C8", "C9", "C10")
+MACHINE_TYPES = (
```

Comment:
Should this list also include GCP VM types? `K80`, `P100`, `GV100`, `GV100x8`, `G1`, `G6`, and `G12`?

Reply:
I don't remember where I got this list from. Is there any document with all machine type lists?